### PR TITLE
Hotfix/1.46.1

### PR DESCRIPTION
**Pull request by tiffwong**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2021-08-11] - v1.46.1
+
+### Changed:
```
Comment (DevDW) [inline CHANGELOG.md:9]: `### Fixed`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
...
+
+- Fix adding ICMP protocol to Firewall rules
+- Fix root device bugs in Linode Config
```
Comment (DevDW) [inline CHANGELOG.md:12]:
```
- Inability to add Firewall rules that use ICMP protocol
- Root Device bugs in Linode Config dialog
- UI bugs for action buttons in dark mode
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2021-08-11] - v1.46.1
+
```
Comment (DevDW) [inline CHANGELOG.md:8]:
```
### Changed
- Copy in Create Firewall and Add Linode to Firewall drawers
```

**Review by DevDW — APPROVED** [review]
🚀

**Comment by bnussman** [comment]
Can we do anything about this? I was hoping the dark mode button fix to fix this 🙁

https://user-images.githubusercontent.com/6440455/129083777-b21816ae-9e70-432a-9737-1ce834b487f5.mov

**Review by bnussman — APPROVED** [review]
All fixes work!

**Pull request merged**